public/languages/ko.json

```diff
@@ -223,37 +223,37 @@
+            "INTRO_1": "이 템플릿 중 하나를 기반으로 새로운 머신러닝 프로젝트를 만들 수 있습니다.",
+            "INTRO_2": "각 템플릿에는 여러분들이 빠르게 머신러닝을 시작할 수 있도록 훈련 데이터가 포함되어 있습니다.",
+            "INTRO_3": "원한다면 훈련 데이터를 수정할 수 있으며, 또는 데이터를 그대로 사용하여 머신러닝 모델을 만들 수 있습니다."
         },
-        "IMPORT": "Import",
+        "IMPORT": "가져오기",
         "MORE": {
-            "TITLE": "Other data sets",
-            "SUMMARY": "What other data sets would you like to see?",
-            "DESCRIPTION": "Have you got an idea for a data set you'd like to see available here? Or do you have a data set that you'd like to contribute? Please get in touch!",
+            "TITLE": "추가 데이터 세트",
+            "SUMMARY": "어떤 데이터 세트를 보고 싶습니까??",
+            "DESCRIPTION": "이 데이터 세트를 사용한 새로운 아이디어가 있습니까? 아니면 기여하고자하는 데이터 세트가 있습니까? 그렇다면 연락하십시오!",
             "MOREINFO": "More..."
         },
         "DATA": {
             "TITANIC": {
-                "TITLE": "Titanic survivors",
-                "SUMMARY": "Predict survival on the Titanic",
-                "DESCRIPTION": "On 15th April 1912, the RMS Titanic hit an iceberg and sank, killing over 1500 of the passengers and crew. This project includes personal data about hundreds of the passengers. You can use this to train a machine learning model that, if you give it a description of a passenger, can learn to predict the likelihood that they would have survived. ",
-                "DETAILS": "The 'ticket class' field is a number indicating the type of ticket for the passenger: first class (1), second class (2), or third class (3). \nThe 'gender' field is whether the passenger was 'male' or 'female'. \nThe 'age' field is the age of the passenger. \nThe 'sibl. sp.' is the number of brothers or sisters (siblings) the passenger had on board if they were a child, or the number of husbands or wives (spouses) the passenger had on board if they were an adult. \nThe 'par. ch.' field is the number of parents the passenger had on board if they were a child, or the number of children the passenger had on board if they were an adult. \nThe 'ticket fare' field is how much they paid for their ticket. \nThe 'embarked' field is where they got on board the Titanic, either Cherbourg, Queenstown or Southampton."
+                "TITLE": "타이타닉호 생존자",
+                "SUMMARY": "타이타닉호 생존자 예측",
+                "DESCRIPTION": "1912 년 4 월 15 일 타이타닉호가 빙산에 충돌한 후 침몰하여 승객과 승무원 중 1500명이 사망하였습니다.이 프로젝트에는 수백 명의 승객에 대한 개인 데이터가 포함되어 있습니다. 이를 사용하여 승객이 살아남을 수 있는 가능성을 예측할 수 있습니다. ",
+                "DETAILS": "'ticket class'필드는 1등석 (1), 2등석 (2) 또는 3등석 (3) 승객의 티켓 종류를 나타내는 번호입니다. \ n 'gender'입력란은 승객이 '남성'또는 '여성'인지 나타냅니다. \ n 'age'필드는 승객의 나이입니다. \ n 'sibl. sp.' 승객이 어렸을 때 동승 한 형제 또는 자매(들)의 수 또는 승객이 성인인 경우 탑승한 남편 또는 아내(배우자)의 수입니다. \ n 'par. ch.' 필드는 어른인 경우 승객이 탑승한 부모의 수 또는 성인인 경우 승객이 탑승한 자녀의 수입니다. \ n 'ticket fare'필드는 티켓 구입에 지불한 금액입니다. n 'embarked'필드는 Cherbourg, Queenstown 또는 Southampton와 같이 타이타닉호에 탑승 한 곳입니다."
```

Comment:
This line is the reason why [the Travis build](https://travis-ci.org/IBM/taxinomitis/builds/563988745#L1344) failed.  

The `\ n` are breaking the file from being valid JSON - please can you remove the space in-between the slash (`\`) and the `n`.

I think it should be this:
```
"DETAILS": "'ticket class'필드는 1등석 (1), 2등석 (2) 또는 3등석 (3) 승객의 티켓 종류를 나타내는 번호입니다. \n 'gender'입력란은 승객이 '남성'또는 '여성'인지 나타냅니다. \n 'age'필드는 승객의 나이입니다. \n 'sibl. sp.' 승객이 어렸을 때 동승 한 형제 또는 자매(들)의 수 또는 승객이 성인인 경우 탑승한 남편 또는 아내(배우자)의 수입니다. \n 'par. ch.' 필드는 어른인 경우 승객이 탑승한 부모의 수 또는 성인인 경우 승객이 탑승한 자녀의 수입니다. \n 'ticket fare'필드는 티켓 구입에 지불한 금액입니다. \n 'embarked'필드는 Cherbourg, Queenstown 또는 Southampton와 같이 타이타닉호에 탑승 한 곳입니다."
```

Reply:
Fixed in https://github.com/IBM/taxinomitis/commit/03c050ea43710cb1aabc888a488649d38a1a22dc